guides/doc-Upgrading_and_Updating/topics/preparing_to_migrate_pulp_content.adoc

```diff
@@ -15,6 +15,30 @@ During migration to Pulp 3, the amount of data stored in the PostgreSQL data dir
 Ensure that you have enough space for the migration.
 A rough estimation of the space you require is 2-3 times the size of the MongoDB Pulp 2 database.
 
+.Pre-Migration Recommendations
+. Ensure that all enabled repositories are synced properly. If any repositories are synced with a *Warning* or *Error* state, fix the underlying problem and resync the repository.
+
+. Ensure no paused tasks in the system are related to any repositories or content views. If there are any:
+
+* Investigate the underlying problem and resume the task to have it completed successfully.
+
+. If a paused content view task is being cleaned up or has been deleted without fixing the underlying cause, ensure that the related broken version of the content view is being deleted as it is already an unusable version.
```

Comment:
```suggestion
* If a paused Content View task is being cleaned up or has been deleted without fixing the underlying cause, ensure that the related broken version of the Content View is being deleted as it is already an unusable version.
```
I've literally no idea what you're saing here so cannot clean it up. 
So many uses of the verb "to be" 

IBM style says never to use "as" - always use "because" to avoid mis-translation